homeassistant/components/daikin/config_flow.py

```diff
@@ -130,6 +130,13 @@ async def async_step_zeroconf(self, discovery_info):
         """Prepare configuration for a discovered Daikin device."""
         _LOGGER.debug("Zeroconf user_input: %s", discovery_info)
         devices = Discovery().poll(ip=discovery_info[CONF_HOST])
+        if not devices:
+            _LOGGER.debug(
+                "Could not find MAC-address for %s,"
+                " make sure the required UDP ports are open (see integration documentation).",
```

Comment:
We don't end logging messages with period.

Reply:
ah right. I fixed it for OP here #38920